README.md

```diff
@@ -1 +1,58 @@
-# ext-slider-widget
\ No newline at end of file
+# ext-slider-widget
+
+This extension provides a image slider widget.
+
+The widget is configurable in regards of amount of slides and items on the slides.
+
+## Configuration
+
+If you want to use the image slider widget, you can create a HTML widget with following content:
+
+```json
+<!--Widget
+{
+  "type": "@shopgate/ext-slider-widget/SliderWidget",
+  "settings": {
```

Comment:
please add also examples for the other supported formats

Reply:
done